cekit/descriptor/env.py

```diff
@@ -6,11 +6,14 @@
 
 env_schema = yaml.safe_load(
     """
-map:
+type: map
+allowempty: True
+mapping:
   name: {type: str, required: True}
   value: {type: any}
   example: {type: any}
-  description: {type: str}"""
+  description: {type: str}
+  metadata: {type: any}"""
```

Comment:
So, in the end we add the `metadata` key as well? I thought we dropped the idea.

Reply:
I guess @jmtd missed that! Let me remove it...

Reply:
Sure, let's blame @jmtd! :)